src/vxingest/builder_common/ingest_manager.py

```diff
@@ -118,9 +118,7 @@ def connect_cb(self):
             _attempts = 0
             while _attempts < 3:
                 try:
-                    self.cluster = Cluster(
-                        "couchbase://" + self.cb_credentials["host"], options
-                    )
+                    self.cluster = Cluster(self.cb_credentials["host"], options)
```

Comment:
Do we need to do any validation to make sure the `couchbase://` or `couchbases://` strings are in the `cb_credentials["host"]` field?